rasa/utils/tensorflow/layers.py

```diff
@@ -10,9 +10,32 @@
 
 
 class SparseDropout(tf.keras.layers.Dropout):
+    """Applies Dropout to the input.
+
+    Dropout consists in randomly setting
+    a fraction `rate` of input units to 0 at each update during training time,
+    which helps prevent overfitting.
+
+    Arguments:
+        rate: Float between 0 and 1; fraction of the input units to drop.
+    """
+
     def call(
-        self, inputs: tf.Tensor, training: Optional[Union[tf.Tensor, bool]] = None
+        self, inputs: tf.SparseTensor, training: Optional[Union[tf.Tensor, bool]] = None
     ) -> tf.Tensor:
+        """Apply dropout to sparse inputs.
+
+        Arguments:
+            inputs: Input sparse tensor (of any rank).
+            training: Python boolean indicating whether the layer should behave in
+                training mode (adding dropout) or in inference mode (doing nothing).
+
+        Returns:
+            Output of dropout layer.
+        """
```

Comment:
Strictly speaking, you also need to add a `Raises` to the doc string if the function raises an exception...